packages/editor/src/utils/badge-properties.tsx

```diff
@@ -10,9 +10,22 @@ const badgePropertyCMS: BadgeType = {
 };
 
 const badgePropertyData: BadgeType = {
-  regex: /#{\s*data\.[^\s}]+\s*}/,
+  regex: /#{\s*(?:data|ivyFormDataTableHandler)\.[^\s}]+\s*}/,
```

Comment:
wouldn't it be much easier to just add a new badge type?
and can you test this?